### PR TITLE
BACKLOG-17368: Fix left alignment of objects

### PR DESCRIPTION
**Pull request by cedmail**
## JIRA

https://jira.jahia.org/browse/BACKLOG-17368

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.expansePanel {
+    padding: 24px 24px 32px 0 !important;
```
Comment (gflores-jahia) [inline src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss:2]: We should probably look at how we can set without using `!important` here. (we can usually override if we use with more specific selectors)

File: src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+}
+
+.button {
+    margin-left: 0 !important;
```
Comment (gflores-jahia) [inline src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss:6]: We should probably look at how we can set without using `!important` here. (we can usually override if we use with more specific selectors)

esp. this one as `.button` looks a bit more generic and not sure if that can affect other buttons

**Review by gflores-jahia — CHANGES_REQUESTED** [review]

File: src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss (hunk 1 above)
Comment (cedmail) [reply src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss:2]: those classes are rewritten/renamed by SCSS compiler so you usually put names that make sense for your local component as there will be no side effect
issue is that moonstone define same properties, material ui also so there is already 2 classes with padding. we need !important to be sure it's this one applying

Comment (gflores-jahia) [reply src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss]: Fixed with https://github.com/Jahia/jcontent/pull/536/commits/12ef5999fdfdeec3027d00dd55e061197788190d

File: src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss (hunk 2 above)
Comment (gflores-jahia) [reply src/javascript/JContent/actions/editImage/ImageEditorDialog/ImageEditorActions/ImageEditorActions.scss]: Fixed with https://github.com/Jahia/jcontent/pull/536/commits/12ef5999fdfdeec3027d00dd55e061197788190d

**Review by gflores-jahia — APPROVED** [review]

**Pull request merged**